packages/core/src/browser/keybinding.ts

```diff
@@ -553,10 +553,10 @@ export class KeybindingRegistry {
                 } else {
                     const command = this.commandRegistry.getCommand(binding.command);
                     if (command) {
-                        const commandHandler = this.commandRegistry.getActiveHandler(command.id);
+                        const commandHandler = this.commandRegistry.getActiveHandler(command.id, binding.args);
 
                         if (commandHandler) {
-                            commandHandler.execute();
+                            commandHandler.execute(binding.args);
```

Comment:
I don't know if that's a common use case, but should a user be able to pass multiple arguments?

```ts
commandHandler.execute(... Array.isArray(binding.args) ? args : [args])
```

Reply:
It is not what happens in VS Code.

Reply:
Fair enough.